examples/modals/modal.py

```diff
@@ -0,0 +1,42 @@
+import nextcord
+
+TESTING_GUILD_ID = 123456798  # Replace with your testing guild id
+
+class Pet(nextcord.ui.Modal):
+    def __init__(self):
+        super().__init__(
+            "Your pet",
+            timeout=5*60, # 5 minutes
+        )
+        self.name = nextcord.ui.TextInput(
+            label="Your pet's name",
+            max_length=50,
+        )
+        self.description = nextcord.ui.TextInput(
+            label="Description",
+            style=nextcord.TextInputStyle.paragraph,
+            placeholder="Information that can help us recognise your pet",
+            required=False,
+            max_length=1800,
+        )
+        self.add_item(self.name)
+        self.add_item(self.description)
+    
+    async def callback(self, inter: nextcord.Interaction) -> None:
+        response = f"{inter.user.mention} favourite pet's name is {self.name.value}."
+        if self.description.value != "":
+            response += f"\nIts pet can be recognized by this information:\n{self.description.value}"
+        await inter.send(response)
+
+bot = nextcord.Client()
```

Comment:
Assuming #495 is merged, all examples will be using commands.Bot instead of nextcord.Client

```suggestion
bot = commands.Bot(command_prefix="$")
```